fapi-client/src/main/scala/com/gu/facia/api/utils/Backfill.scala

```diff
@@ -0,0 +1,55 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.GuardianContentClient
+import com.gu.facia.api.contentapi.ContentApi
+import com.gu.facia.api.contentapi.ContentApi._
+import com.gu.facia.api.models.{CollectionConfig, CuratedContent, FaciaContent}
+import com.gu.facia.api.{FAPI, Response}
+import com.gu.facia.client.ApiClient
+import com.gu.facia.client.models.{Backfill, TrailMetaData}
+
+import scala.concurrent.ExecutionContext
+
+case class InvalidBackfillConfiguration(msg: String) extends Error(msg)
+
+object BackfillContent {
+  def resolveFromConfig(collectionConfig: CollectionConfig): BackfillResolver = {
```

Comment:
This could now go into `BackfillResolver`!

Is the name `BackfillContent` better than `BackfillResolver`?